Response/ExceptionResponse.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace OAT\Bundle\Lti1p3Bundle\Response;
+
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+final class ExceptionResponse extends Response
```

Comment:
do you need to introduce a full mew (and final, not ideal in a lib/bundle) class when at the end you just do a `htmlspecialchars()` ?

Reply:
By changing the first file, it includes the code in one method but now we have the same problem in two different classes. I don't see the problem in creating a new class with **Single Responsibility**.

Reply:
the class with such responsibility already exists, it's SF Response :)

here you should just build one with htmlspecialchars() the body, imo no need for such complexity

Reply:
this is not resolved :)

Reply:
I agree here. Let's remove that additional response implementation.